packages/guideline-blocks-settings/src/components/RichTextEditor/serializer/serializeToHtml.ts

```diff
@@ -9,7 +9,7 @@ export const serializeRawToHtmlAsync = async (
     raw: string,
```

Comment:
There shouldn't be any changes in here, prettier ;)

Reply:
Fixed, and all the other prettier issues as well 😇